literals/obfuscators/obfuscators.go

```diff
@@ -0,0 +1,60 @@
+package obfuscators
+
+import (
+	cryptrand "crypto/rand"
+	"fmt"
+	"go/ast"
+	"go/token"
+	mathrand "math/rand"
+	"os"
+	"strings"
+)
+
+// Obfuscator takes a byte slice and converts it to a ast.BlockStmt
+type Obfuscator interface {
```

Comment:
any reason this is a separate package from literals? because the name here ends up being `obfuscators.Obfuscator`, which is very repetitive, whereas `literals.Obfuscator` is much better. It also doesn't seem like there's enough code to warrant two library packages here.

Reply:
You could have separate filenames, no? A thousand lines seems completely fine for a single package. Or if you mean for the exposed API, I assume it should be small either way; just the interface, the implementations, and little more.